src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyCodeFixProvider.cs

```diff
@@ -85,6 +87,19 @@ public sealed override Task RegisterCodeFixesAsync(CodeFixContext context)
 
     private async Task<Solution> ProcessResultAsync(
         Solution originalSolution, Solution currentSolution, Diagnostic diagnostic, CancellationToken cancellationToken)
+    {
+        try
+        {
+            return await ProcessResultWorkerAsync(originalSolution, currentSolution, diagnostic, cancellationToken).ConfigureAwait(false);
+        }
+        catch (Exception ex) when (FatalError.ReportAndCatchUnlessCanceled(ex))
```

Comment:
heard from community member about things occasionally failing.  adding some dedicated NFWs for this.